https://issues.redhat.com/browse/ACM-27403 MCOA known issue (#8449)
open-cluster-management/rhacm-docs: release_notes/known_issues_observability.adoc

For more about deprecations and removals, see xref:../release_notes/acm_deprecate_remove.adoc#deprecations-removals-acm[Deprecations and removals for {acm-short}]. 

[#mcoa-fails]
== Multicluster observability add-on fails to collect metrics

> The `multicluster-observability-addon-manager` pod is in a `OOMKilled` state because there are too many resources being received by the pod. By default, the `multicluster-observability-addon-manager` pod is configured to watch all config maps and secrets on the hub cluster. When you enable the multicluster observability add-on, the metric collection fails. 

Reviewer: ```suggestion
The `multicluster-observability-addon-manager` pod is in a `OOMKilled` state because the pod watches all config maps and secrets on the hub cluster, causing it to use too many resources. When you enable the multicluster observability add-on, the metric collection fails. 
```
